src/main/java/pl/cleankod/exchange/entrypoint/configuration/ControllersLoggingAspect.java

```diff
@@ -0,0 +1,31 @@
+package pl.cleankod.exchange.entrypoint.configuration;
+
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.AfterThrowing;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+
+import java.util.logging.Logger;
+
+@Aspect
+public class ControllersLoggingAspect {
```

Comment:
What would be the alternatives to using aspects? Maybe Spring has a built-in feature for that? Or maybe there is a library we could use?

Reply:
There is a thing called `CommonsRequestLoggingFilter` that could be used instead of aspects. 
Replaced aspect with CommonsRequestLoggingFilter